stetho/src/main/java/com/facebook/stetho/inspector/database/SqliteDatabaseDriver.java

```diff
@@ -206,18 +212,28 @@ private static String getFirstWord(String s) {
     return handler.handleRawQuery();
   }
 
-  private SQLiteDatabase openDatabase(String databaseName) throws SQLiteException {
-    Util.throwIfNull(databaseName);
-    return mDatabaseConnectionProvider.openDatabase(findDatabaseFile(databaseName));
+  private SQLiteDatabase openDatabase(
+      SqliteDatabaseDescriptor databaseDesc)
+      throws SQLiteException {
+    Util.throwIfNull(databaseDesc);
+    return mDatabaseConnectionProvider.openDatabase(databaseDesc.file);
   }
 
-  private File findDatabaseFile(String databaseName) {
-     for (File providedDatabaseFile : mDatabaseFilesProvider.getDatabaseFiles()) {
-       if (providedDatabaseFile.getName().equals(databaseName)) {
-         return providedDatabaseFile;
-       }
-     }
+  static class SqliteDatabaseDescriptor implements DatabaseDescriptor {
+    public final File file;
+
+    public SqliteDatabaseDescriptor(File file) {
+      this.file = file;
+    }
 
-     return mContext.getDatabasePath(databaseName);
-   }
+    @Override
+    public String context() {
```

Comment:
Discussed with @longinoa, this approach is misleading and kludgey.  Will follow up with using a namespace / tag field on the driver itself and disambiguating when multiple drivers are in use.